python/xoscar/context.pyx

```diff
@@ -259,6 +290,16 @@ cdef class ClientActorContext(BaseActorContext):
         context = self._get_backend_context(address)
         return context.get_pool_config(address)
 
+    def buffer_ref(self, str address, buf: Any) -> BufferRef:
+        context = self._get_backend_context(address)
+        return context.buffer_ref(address, buf)
+
+    def copy_to(self, local_buffers: List[bytes], remote_buffer_refs: List[BufferRef], block_size: Optional[int] = None):
+        if remote_buffer_refs:
```

Comment:
Why check this?

Reply:
Fix. If any of ``local_buffers`` and ``remote_buffer_refs `` is empty, just raise error.